docker-compose.yml

```diff
@@ -125,6 +118,11 @@ services:
     volumes:
       # Logs
       - logs:/var/log/promtail
+    healthcheck:
+      test: "curl -f localhost:4001/livez && curl -f localhost:4001/readyz"
+      interval: 3s
+      timeout: 1s
+      retries: 10
     depends_on:
       postgres:
         condition: service_healthy
```

Comment:
Do we not have health checks for the trigger-service?

Reply:
Added HTTP healtcheck with curl